exampleCourse/questions/element/dragDropBlocks/question.html

```diff
@@ -0,0 +1,108 @@
+  <code>max-incorrect="4"</code> to display a minimum of 2 incorrect distractors and maximum of 4.</p>
+
+</pl-question-panel>
+<pl-order-blocks answers-name="prime-numbers-ordered" shuffle-options="true" min-incorrect="2" max-incorrect="4">
+  <pl-answer correct="false">1</pl-answer>
+  <pl-answer correct="true">2</pl-answer>
+  <pl-answer correct="true">3</pl-answer>
+  <pl-answer correct="false">4</pl-answer>
+  <pl-answer correct="true">5</pl-answer>
+  <pl-answer correct="false">6</pl-answer>
+  <pl-answer correct="true">7</pl-answer>
+  <pl-answer correct="false">8</pl-answer>
+  <pl-answer correct="false">9</pl-answer>
+  <pl-answer correct="false">10</pl-answer>
+</pl-order-blocks>
+
+<hr>
+
+<pl-question-panel>
+<p>
+  <strong>Example 3:</strong>  Implement the Python function <code>my_sum</code>, which takes 2 numbers, <code>first</code> and <code>second</code>,
+  and return their sum. Note that the ordering of instantiation, computation, and returning of the value must be correct.
+  Remember to indent your code! </p>
+<p>
+  This example uses the attribute <code>pl-hide-in-panel answer="true"</code> to hide the correct answer. It
+  also uses <code>permutation-mode="ranking"</code> to control how the correct solution is ordered
+  and <code>indent</code> attribute in <code>pl-answer</code> to define expected code indentation.
+</p>
+</pl-question-panel>
+
+<pl-hide-in-panel answer="true">
+<pl-order-blocks answers-name="python-ordered" shuffle-options="true"  permutation-mode="ranking">
+  <pl-answer correct="true" ranking="1" indent="1">def my_sum(first, second):</pl-answer>
+  <pl-answer correct="true" ranking="2" indent="2">sum = 0</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="2">sum += first</pl-answer>
+  <pl-answer correct="true" ranking="3" indent="2">sum += second</pl-answer>
+  <pl-answer correct="true" ranking="4" indent="2">return sum</pl-answer>
+  <pl-answer correct="false">sum + first</pl-answer>
+  <pl-answer correct="false">sum + second</pl-answer>
+  <pl-answer correct="false">This is an example very long tile to test CSS overflow</pl-answer>
+
+</pl-order-blocks>
+</pl-hide-in-panel>
+
+
+<hr>
+
+
+<pl-question-panel>
+<p>
+  <strong>Example 4:</strong> Let $a, b$ be any even numbers. Prove that $a + b$ is always even.
+</p>
+
+<p>This example uses the attribute <code>permutation-mode="ranking"</code> to control how the correct solution is ordered.
+When the order of a correct block is not relevant,
+ the attribute <code>ranking</code> is not provided. In this example, two of the blocks can be placed at any
+position when constructing the solution. The attribute <code>dropzone-layout="verticalLayout"</code> is used
+to display the input area above the output area.</p>
+</pl-question-panel>
+
+ <pl-order-blocks answers-name="latex_answers" shuffle-options="true" permutation-mode="ranking" dropzone-layout="verticalLayout">
```

Comment:
@mwest1066 my suggestion is that we merge the PR as is, and work on the improvement of the indentation in a future PR. I also had the same impression that this should be improved. Is it okay to add this improvement for a future PR?

Reply:
I think we should add an option now to activate indentation (default false). I don't want to leave this for later because that would cause a change in behavior (indentation would suddenly stop working). We don't need to add the help text now.

Reply:
@mwest1066  as of now `check-indentation` basically grades the solution for given indentation. However, it does not control the possibility of blocks being dragged with indentation or not. Do we want to have these two features separate, or combined?

For example, do we want to have an attribute `allow-solution-indentation` that allows the solution to be dragged at different position (true) or all flushed left (false) and another attribute `grade-indentation` that will check for the child attribute `indent`?

Or should we just assume that if not checking for indentation during grading, then we should not allow the indentation to happen anyway in the output area? In that case, we only need one attribute (let's say `allow-solution-indentation`) that allows blocks to be placed with indentation, and will be searching for the attribute `indent` in `pl-answer`. 

I think this last option makes more sense, but just want to confirm, since the last comment suggests a different attribute.

Reply:
I agree that we probably only need one attribute. Maybe just `indentation` (boolean, default false)? If this is false then it would not be possible to drag blocks left/right to control indentation, so of course we would also not check the indentation for correctness. If `indentation=true` then we would allow the blocks to be indented, and we would check this (and in a future PR we would also show help text explaining this).

Reply:
@mwest1066 I think I see one issue if we combine the UI allow and grading into one attribute. If we want to use the external grading, the UI indentation should be allowed, however, the indent attribute is not used to check for grading.

we can still use only one attribute, but that will require to explicitly set indentation = "true" when using the external grader (but ignoring the indent from child element). what do you think?

Reply:
this has been added, although the option to add a help text at the bottom specifying indention is expected is WIP. This item has been added to #3476